packages/canvas-tokens/utils/transformers/flatRGBAColor.ts

```diff
@@ -13,7 +13,7 @@ export const flatRGBAColor = ({value}: DesignToken): string => {
     const innerRgb = b.replace(/rgba\(([^}]+)/g, (__: string, c: string) =>
       c.split(',').slice(0, 3).toString()
     );
-    return `rgba(${innerRgb},${alpha})`;
+    return alpha === '0' ? 'transparent' : `rgba(${innerRgb},${alpha})`;
```

Comment:
Comment: I wonder if we should check for 0 or 0% here. 🤔

Reply:
It's not looking for opacity with % at all. I'm suggesting do not use percentage in color, as we using regular tokens

Reply:
I've added support for this as well